client/src/components/TypeModal.jsx

```diff
@@ -25,26 +22,29 @@ class TypeModal extends Component {
   }
 
   addType() {
+    // is there a way to destructure this function from props?
     this.props.addNewType(this.state.type);
   }
```

Comment:
Is there a best practice about destructing? I destructure state in the render method below. I also am wondering if I could destructure the function I call on line 26 above.